src/ui/processing/qgsprocessingmeshdatasettimewidget.ui

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>QgsProcessingMeshDatasetTimeWidgetBase</class>
+ <widget class="QWidget" name="QgsProcessingMeshDatasetTimeWidgetBase">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>465</width>
+    <height>122</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string notr="true">Form</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout">
+   <item>
+    <layout class="QHBoxLayout" name="horizontalLayout">
+     <item>
+      <widget class="QRadioButton" name="radioButtonCurrentCanvasTime">
+       <property name="text">
+        <string>Current Canvas Time</string>
```

Comment:
```suggestion
        <string>Current canvas time</string>
```
Check box labels should be capitalized just like standard labels -- i.e. sentence case